src/main/java/com/dev/cinema/model/ShoppingCart.java

```diff
@@ -0,0 +1,28 @@
+package com.dev.cinema.model;
+
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table (name = "shopping_carts")
+public class ShoppingCart {
+    @Id
+    private Long id;
+    @OneToMany(fetch = FetchType.EAGER)
```

Comment:
Do we need the EAGER fetch type in here?

Reply:
let's remove eager fetching - in general, this is considered bad practice
and find another way to get tickets along with the shopping cart

Reply:
fixed